vsts/scripts/dockerCleanup.sh

```diff
@@ -38,20 +38,29 @@ docker images
 #
 # **NOTE**
 # - We still keep the tags of the following pattern because we still need some cache so that next builds are faster
-#	a. oryxdevms/build:latest
-#	b. oryxdevms/python-<major.minor>:latest
-#	b. oryxdevms/node-<major.minor>:latest
-#	b. oryxdevms/dotnetcore-<major.minor>:latest
+#	a. oryx/build:latest
+#	b. oryx/python-<major.minor>:latest
+#	c. oryx/node-<major.minor>:latest
+#	d. oryx/dotnetcore-<major.minor>:latest
 # - We should untag these images only after they have been pushed to a remote repository.
+# Untag images with DockerHub related tags
+UntagImages "oryxdevms/*:latest"
 UntagImages "oryxdevms/*:*.*"
-UntagImages "oryxtests/*:latest"
 UntagImages "oryxprod/*:latest"
 UntagImages "oryxprod/*:*.*"
+
+# Untag images with Dev ACR related tags
 UntagImages "oryxdevmcr.azurecr.io/public/oryx/*:latest"
 UntagImages "oryxdevmcr.azurecr.io/public/oryx/*:*.*"
+
+# Untag images with Staging ACR related tags
```

Comment:
This is required since "Release" related stuff pulls in images onto the build agent.